content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -311,19 +311,18 @@ assuming that the number of replicas is not also changed).
 
 ### Scaling a ReplicaSet
 
-A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
+You can scale a ReplicaSet up or down by updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
-When scaling down, the ReplicaSet controller chooses which pods to delete by sorting the available pods to
-prioritize scaling down pods based on the following general algorithm:
+When scaling down, the ReplicaSet controller chooses which pods to delete based on the following general algorithm:
 
-1. Pending (and unschedulable) pods are scaled down first
+1. Pending and unschedulable pods are scaled down first.
 1. If `controller.kubernetes.io/pod-deletion-cost` annotation is set, then
-   the pod with the lower value will come first.
-1. Pods on nodes with more replicas come before pods on nodes with fewer replicas.
+   the pod with the lower value is deleted first.
+1. Pods on nodes with more replicas are deleted before pods on nodes with fewer replicas.
 1. If the pods' creation times differ, the pod that was created more recently
-   comes before the older pod (the creation times are bucketed on an integer log scale
-   when the `LogarithmicScaleDown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled)
+   is deleted before the older pod. The creation times are bucketed on an integer log scale
+   when the `LogarithmicScaleDown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
```

Comment:
```suggestion
1. The ReplicaSet controller prefers to delete the Pods that have been running for
   the **least** amount of time. When the `LogarithmicScaleDown`
   [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is
   enabled in your cluster, the ReplicaSet controller does not use the exact age of
   the Pod. Instead, the controller calculates a score for how old the Pods in the
   ReplicaSet are, using a logarithmic bucketing algorithm. The ReplicaSet controller
   then looks for the lowest score from that algorithm, and selects at random one Pod
   with that score.
```

Reply:
You're welcome to make equivalent changes locally or via the web editor.

Reply:
If you'd like more help, join Slack - see https://slack.k8s.io/ - and say Hi in the #sig-docs channel.